docs/resources/eox-tagging.postman_collection.json

```
@@ -0,0 +1,440 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{domain}}/eox-tagging/api/v1/tags/",
+					"host": [
+						"{{domain}}"
+					],
+					"path": [
+						"eox-tagging",
+						"api",
+						"v1",
+						"tags",
+						""
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "5. Tags - Course Overview",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"var jsonData = JSON.parse(responseBody);",
+							"pm.environment.set(\"courseoverview_tag_key\", jsonData.key);"
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "JWT {{token}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n   \"tag_value\": \"premium\",\n   \"tag_type\": \"subscription_level_course\",\n   \"access\": \"public\",\n   \"activation_date\": \"2021-05-10T20:50:04.934Z\",\n   \"expiration_date\": \"2025-05-10T20:50:04.934Z\",\n   \"target_type\": \"courseoverview\",\n   \"target_id\": \"{{course_id}}\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{domain}}/eox-tagging/api/v1/tags/",
+					"host": [
+						"{{domain}}"
+					],
+					"path": [
+						"eox-tagging",
+						"api",
+						"v1",
+						"tags",
+						""
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "6. Tags",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							""
+						],
+						"type": "text/javascript",
+						"packages": {}
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "JWT {{token}}",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{domain}}/eox-tagging/api/v1/tags/",
+					"host": [
+						"{{domain}}"
+					],
+					"path": [
+						"eox-tagging",
+						"api",
+						"v1",
+						"tags",
+						""
+					],
+					"query": [
+						{
+							"key": "key",
+							"value": "{{user_tag_key}}",
+							"disabled": true
+						},
+						{
+							"key": "key",
+							"value": "{{courseenrollment_tag_key}}",
+							"disabled": true
+						},
+						{
+							"key": "key",
+							"value": "{{courseoverview_tag_key}}",
+							"disabled": true
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "7. Tags - User",
+			"request": {
+				"method": "DELETE",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "JWT {{token}}",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{domain}}/eox-tagging/api/v1/tags/{{user_tag_key}}",
```

Without the slash it doesn't work
```suggestion
					"raw": "{{domain}}/eox-tagging/api/v1/tags/{{user_tag_key}}/",
```